src/backend/src/routes/reimbursement-requests.routes.ts

```diff
@@ -79,4 +79,10 @@ reimbursementRequestsRouter.post(
   ReimbursementRequestController.createExpenseType
 );
 
+reimbursementRequestsRouter.delete(
+  '/:requestId/delete',
+  validateInputs,
```

Comment:
you don't need to call validate inputs since you're not actually validating anything in the route